noria/src/view.rs

```diff
@@ -59,6 +59,7 @@ pub enum ReadReply {
 #[doc(hidden)]
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct ViewBuilder {
+    pub name: String,
```

Comment:
Why do views now need to carry their own name? I don't think it's a bad change, just curious.

Reply:
I think it might make sense for `ViewBuilder` to know which replica (i.e., `usize`) it is interacting with, especially when we later add fail-over. Maybe just add that field?

Reply:
👍